Sources/SPMBuildCore/BuildSystem/BuildSystem.swift

```diff
@@ -168,6 +168,14 @@ public struct BuildSystemProvider {
     }
 }
 
+extension BuildSystemProvider.Kind {
+    public var useXcodeBuildSystemPath: Bool {
```

Comment:
suggestion: is this not just a Build System path. Maybe update the variable name to `useXcodeBuildEngine`

Reply:
Done